server/models/registration/error.go

```diff
@@ -0,0 +1,82 @@
+package registration
+
+import (
+	"fmt"
+
+	"github.com/layer5io/meshkit/errors"
+)
+
+const (
+	ErrDirPkgUnitParseFailCode = "replace_me"
+	ErrGetEntityCode = "replace_me"
+	ErrRegisterEntityCode = "replace_me"
+	ErrImportFailureCode = "replace_me"
+	ErrMissingRegistrantCode = "replace_me"
+	ErrSeedingComponentsCode = "replace-me"
+)
+
+func ErrSeedingComponents(err error) error {
+	return errors.New(
+		ErrSeedingComponentsCode,
+		errors.Alert,
+		[]string{"Failed to register the given models into meshery's registry"},
+		[]string{err.Error()},
+		[]string{"Given models may not be in accordance with Meshery's schema", "Internal(OS level) error while reading files" },
+		[]string{"Make sure the models being seeded are valid in accordance with Meshery's schema", "If it is an internal error, please try again after some time"},
+	)
+}
+
+func ErrMissingRegistrant() error {
+	return errors.New(
+		ErrMissingRegistrantCode,
+		errors.Alert,
+		[]string{"Meshery models are always registered in context of a registrant"},
+		[]string{""},
+		[]string{"Given model might not contain information about the registrant"},
```

Comment:
We can include the modelName so that we are specific about which model lacks the registrant info.